qemu/tests/uefishell.py

```diff
@@ -211,6 +220,11 @@ def form_ping6_args():
             func_name = params["command_%s_%s" % (scenario, "args")]
             command += eval(func_name)
         check_result = params.get("check_result_%s" % scenario)
-        uefishell_test.send_command(command, check_result, time_interval)
+        output = uefishell_test.send_command(command,
+                                             check_result,
+                                             time_interval)
+        if params.get("handle_%s" % scenario):
+            func_name = params["handle_%s" % scenario]
+            eval(func_name)(output[0])
```

Comment:
what about passing the whole 'output' list to the handler function, for you might have to handle several matched results in future.

Reply:
agree with you, thanks